### ISSUE TITLE
A sitemap plugin ?

### ISSUE THREAD

**Issue by oblab** [sebsauvage/Shaarli#250]
I'm not a php developper, but it would be useful to have a plugin that generate a sitemap.xml for Shaarli.
I saw this attempt https://github.com/sebsauvage/Shaarli/pull/200/files
Any idea to do this on Shaarli  0.12.1 ?

**Comment by ArthurHoaro** [comment]
Please look for solutions or open your issue on github.com/shaarli/Shaarli instead.

**Comment by oblab** [comment]
Oui, j'ai vu que je m'étais trompé d'espace, j'ai déplacé ma requête. Merci ;)

**Comment by e2jk** [comment]
@oblab please close this issue, as you mentioned/realized the project lives on at https://github.com/shaarli/Shaarli
